Libraries/Microsoft.Teams.Plugins/Microsoft.Teams.Plugins.AspNetCore/AspNetCorePlugin.Stream.cs

```diff
@@ -34,44 +48,44 @@ public class Stream : IStreamer
         private DateTime? _closedAt;
         private int _count = 0;
         private MessageActivity? _result;
-        private readonly SemaphoreSlim _lock = new(1, 1);
+        private bool _flushing = false;
```

Comment:
NIT: is it common for private variables to have _ in C#? I know that's common in like C++...